### PR TITLE
[FIVE-189]GR: Crear UnitTest parar el controlador de Projectos

### PR DESCRIPTION
**Pull request by gonzalorom**
Se debe crear la clase de UnitTest para el controlador de Proyectos.

Cambios realizados:

- Se creó la clase ProjectControllerTests con los tests correspondientes al ProjectController.

### CODE REVIEW

File: FiveRockingFingers/FRF.Web/Controllers/ProjectsController.cs [hunk 1]
```diff
@@ -47,15 +47,15 @@ public async Task<IActionResult> GetAll(string userId)
...
-            
-                        var projectsDto = _mapper.Map<IEnumerable<ProjectDto>>(projects);
-                        return Ok(projectsDto);*/
-            return NotFound();
+            // TODO: AWS Credentials, Loggin bypassed.Uncomment after do:
+            //var currentUserId = await _userService.GetCurrentUserId();
+            var currentUserId = "";
+            var projects = await _projectService.GetAllAsync(currentUserId);
+
+            if (projects == null) return StatusCode(204);
```
Comment (matiasbeckerle) [inline FiveRockingFingers/FRF.Web/Controllers/ProjectsController.cs:55]: This should be NotFound. But it also depends... is that service capable of returning a null? Or an empty list?

File: test/FRF.Web.Tests/Controllers/ProjectsControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+            Assert.Equal(project.Id, returnValue.Id);
+            Assert.Equal(project.Name, returnValue.Name);
+            Assert.Equal(project.Budget, returnValue.Budget);
+
+            _projectsService.Verify(mock => mock.GetAsync(projectId), Times.Once);
+            _mapper.Verify(mock => mock.Map<ProjectDto>(It.Is<Project>(c => c.Name == project.Name)), Times.Once);
+        }
+
+        [Fact]
+        public async Task Get_NotFound_WhenProjectIsNotFound()
```
Comment (matiasbeckerle) [inline test/FRF.Web.Tests/Controllers/ProjectsControllerTests.cs:67]: `Get_WhenProjectDoesNotExist_ReturnsNotFound`

Syntax is

MethodUnderTest_(Condition)_Returns

You will need to update more tests 👇

**Review by matiasbeckerle — COMMENTED** [review]
Nice! Left a question.

File: FiveRockingFingers/FRF.Web/Controllers/ProjectsController.cs (hunk 1 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.Web/Controllers/ProjectsController.cs:55]: Yes, this will be NotFound. This improvement is on this pull request #30

Comment (matiasbeckerle) [reply FiveRockingFingers/FRF.Web/Controllers/ProjectsController.cs:55]: Ok, but let's talk next week about null vs empty lists and 404.

**Review by matiasbeckerle — APPROVED** [review]

File: test/FRF.Web.Tests/Controllers/ProjectsControllerTests.cs (hunk 2 above)
Comment (gonzalorom) [reply test/FRF.Web.Tests/Controllers/ProjectsControllerTests.cs]: Done.

**Pull request merged**